openwisp_utils/admin_theme/static/admin/js/menu.js

```diff
@@ -14,8 +14,17 @@ var MenuTransitionTime = '0.1s';
   initAccountViewHandler();
   initToolTipHandlers();
   initResizeScreenHelpers();
+  cleanUp();
 })();
 
+function cleanUp(){
+  // remove dublicate of objects-tools
+  var dublicateObject = document.querySelector('#content-main .object-tools');
+  if(dublicateObject){
+    dublicateObject.remove();
+  }
+}
+
```

Comment:
What is this?

Reply:
Do you remember the overlapping issue of buttons and text on the device detail page.
To fix this I changed the position of ``object-tools``  but still it was being created twice by the django so these lines are used to remove the second one which was causing a repetition of some ids.

https://github.com/openwisp/openwisp-utils/blob/185b72ea675908aca8d12d53eede3dc03ef1bf47/openwisp_utils/admin_theme/templates/admin/base.html#L75-L85

Earlier it was inside ``block content``

Reply:
It seems to me we should solve the issue which is causing this to be duplicated rather than adding code which removes the duplicates. Why is it duplicated in the first place? I suggest to create an issue for this.

Reply:
ok :+1: 
https://github.com/openwisp/openwisp-utils/issues/235